src/pageLayout/components/RenamablePageTitle.scss

```diff
@@ -8,6 +8,7 @@ $renamable-page-title--padding: 0;
 
 .renamable-page-title {
   width: 100%;
+  max-width: 850px;
```

Comment:
if you remove the max width and instead add:

```css
display: inline-block;
```

does that fix the issue?

Reply:
let me see what that does

Reply:
no it results in it going back to the old style